tool/dumper/vkgcPipelineDumper.cpp

```diff
@@ -419,12 +419,16 @@ void PipelineDumper::dumpResourceMappingNode(const ResourceMappingNode *userData
 // Dumps pipeline shader info to file.
 //
 // @param shaderInfo : Shader info of specified shader stage
+// @param dumpDir : Directory of pipeline dump
 // @param [out] dumpFile : Dump file
-void PipelineDumper::dumpPipelineShaderInfo(const PipelineShaderInfo *shaderInfo, std::ostream &dumpFile) {
+void PipelineDumper::dumpPipelineShaderInfo(const PipelineShaderInfo *shaderInfo, const char *dumpDir,
+                                            std::ostream &dumpFile) {
   const ShaderModuleData *moduleData = reinterpret_cast<const ShaderModuleData *>(shaderInfo->pModuleData);
   auto moduleHash = reinterpret_cast<const MetroHash::Hash *>(&moduleData->hash[0]);
 
   // Output shader binary file
+  DumpSpirvBinary(dumpDir, &moduleData->binCode, moduleHash);
```

Comment:
- I think we needn't dump spirv binary again in dumpPipelineShaderInfo because we have do it in BuildShaderModule
- and since we needn't dump spirv binary, we needn't add dumpDir in dumpPipelineShaderInfo